CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Fixed
```

Comment:
It's unfortunate that there's already a duplicated section, no need to repeat the mistake again.

Reply:
Oh yeah lol I have fixed that.